eng/testing/xunit/xunit.targets

```diff
@@ -1,11 +1,5 @@
 <Project>
   <PropertyGroup>
-    <TargetOSCategory Condition="'$(TargetOS)' == 'Windows_NT'">nonwindowstests</TargetOSCategory>
-    <TargetOSCategory Condition="'$(TargetOS)' == 'Linux'">nonlinuxtests</TargetOSCategory>
-    <TargetOSCategory Condition="'$(TargetOS)' == 'OSX'">nonosxtests</TargetOSCategory>
-    <TargetOSCategory Condition="'$(TargetOS)' == 'FreeBSD'">nonfreebsdtests</TargetOSCategory>
-    <TargetOSCategory Condition="'$(TargetOS)' == 'NetBSD'">nonnetbsdtests</TargetOSCategory>
```

Comment:
Could these be removed from https://github.com/dotnet/runtime/blob/952eecabbdd57540f4fe72c490c794e79c3d0cf3/src/installer/test/Directory.Build.props?
(that will close https://github.com/dotnet/arcade/issues/4105 🙂)

Reply:
Unfortunately not. In installer we don't use XUnitExtensions afaik.

Reply:
`XUnitExtensions` are used in installers: https://github.com/dotnet/runtime/blob/7abe42dc1123722ed385218268bb9fe04556e3d3/src/installer/test/TestUtils/TestUtils.csproj#L17 e.g.`PlatformSpecificAttribute` is used here: https://github.com/dotnet/runtime/blob/7abe42dc1123722ed385218268bb9fe04556e3d3/src/installer/test/Microsoft.NET.HostModel.Tests/Microsoft.NET.HostModel.AppHost.Tests/AppHostUpdateTests.cs#L170

Reply:
Yeah just noticed that :), thanks: https://github.com/ViktorHofer/runtime/commit/27048a24ecf74132f1114dd82b8f9e0171fee71c